source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_settlement_report_latest_calculations_v1.sql

```diff
@@ -0,0 +1,5 @@
+CREATE VIEW IF NOT EXISTS {SETTLEMENT_REPORT_DATABASE_NAME}.latest_calculations_v1 as
+SELECT calculation_type,
+       MAX(version) as version
+FROM {BASIS_DATA_DATABASE_NAME}.calculations
+GROUP BY calculation_type
```

Comment:
```suggestion
GROUP BY calculation_type
WHERE calculation_type IN ('BalanceFixing', 'WholesaleFixing', 'FirstCorrectionSettlement', 'SecondCorrectionSettlemen', 'ThirdCorrectionSettlement')
```
(or calculation_type <> 'Aggregation')